src/store/Reducers/GlobalFilterStore.js

```diff
@@ -1,26 +1,34 @@
 import * as ActionTypes from '../ActionTypes';
 
-export const GlobalFilterStore = (state = {}, action) => {
+const initialState = {
+    selectedGlobalTags: [],
+    selectedTags: [],
+    systemProfile: undefined
+};
+
+export const GlobalFilterStore = (state = initialState, action) => {
+    let newState = { ...state };
```

Comment:
This isn't really needed, or am I missing something?

Reply:
Hmm, maybe it is really not needed. But the intention of this line was to avoid mutation of the initial state accidentally by creating a copy of the state inside reducer. Do you think that it makes sense or should I remove it. We are using this in all reducers.

Reply:
Hm. I don't think using `state` directly below and spread it would mutate it or `initialState`. Unless I'm missing something. 

I think, and I'm not sure 100%, that a case where the `initialState` could be mutated is when this line would just be `let newState = state`, but initialState is a `const`, so something would say that it's not possible. And even with that line there would need to be and action that tried to assign or mutate `newState`.

Reply:
Good, I have adopted your suggestion.